firmware/boron-ins-fsm/src/BraveSensorProductionFirmware.ino

```diff
@@ -11,7 +11,7 @@
 #include "tpl5010watchdog.h"
 
 #define DEBUG_LEVEL            LOG_LEVEL_INFO
-#define BRAVE_FIRMWARE_VERSION 9060   // see versioning notes in the readme
+#define BRAVE_FIRMWARE_VERSION 3898   // see versioning notes in the readme
```

Comment:
This change isn't actually related to the feature that you're implementing, so before this PR is merged, could you please change it back to 9060?